app/views/application/_navigation.html.erb

```diff
@@ -20,7 +20,9 @@
       <!-- Right side -->
       <div class="header-right header-menu">
         <span class="header-item">
-          <%= image_tag gravatar_url(current_user.email), class: "img img-circle avatar" %>
+          <%= link_to new_two_step_verification_path do %>
```

Comment:
It's "hidden" on purpose :-) I first want to run it on our "own" server, and think of a proper Account settings page :-)